.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Checklist:
+
+<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->
+
+- [ ] I have read [contribution guidelines](https://github.com/maplelabs/github-audit/blob/master/CONTRIBUTING.md).
+- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
+- [ ] I have tested these changes locally on my machine.
+
+##### SUMMARY
+<!--- Describe the change below, including rationale and design decisions -->
+
+<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
```

Comment:
For all PRs, can we make it mandatory to add git issue id?

Reply:
Features, doc changes, new testcases do not have issue ID.